app/controllers/publishers/vacancies/build_controller.rb

```diff
@@ -88,28 +94,4 @@ def update_vacancy
   def set_steps
     self.steps = step_process.steps - [:review]
   end
-
-  # rubocop:disable Metrics/AbcSize
-  def redirect_to_next_step
```

Comment:
Now we don't need to redefine the redirection in this controller. As it the contact details and confirmation steps/forms can be trated as any other without custom redirection logic.